circle.yml

```diff
@@ -1,17 +0,0 @@
-machine:
-  node:
-    version: 6
-  environment:
-    PATH: "${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
-
-dependencies:
-  override:
-    - yarn
-    - yarn global add bower
-    - bower cache clean && bower install
```

Comment:
Looks like we are force cleaning bower cache here (I think CircleCI automatically caches the entire `~/.cache/` folder). I am wondering if we should perhaps not cache the bower packages. 

Maybe in CircleCI 1.0 we force cleaned cache because we couldn't link it with the `bower.json` file like we can in CircleCI 2.0. I will defer this decision to @alexgb